limiter/limiter.go

```diff
@@ -33,7 +34,7 @@ import (
 // a group from some on-disk database, so you don't have to have all group
 // limits in memory. (Limiter itself will clear out unused groups from its own
 // memory.)
-type SetLimitCallback func(name string) int
+type SetLimitCallback func(context.Context, string) int
```

Comment:
Why is context passed in to many methods of limiter now? I don't see it being used in the diff.